rust/lance-file/src/v2/reader.rs

```diff
@@ -313,10 +313,7 @@ impl ReaderProjection {
 
 #[derive(Clone, Debug, Default)]
 pub struct FileReaderOptions {
```

Comment:
Not related to your PR but let's put a basic docstring on this type since is is part of public API?

Reply:
Added.